test/integration/scheduler/framework_test.go

```diff
@@ -579,29 +586,46 @@ func TestPostFilterPlugin(t *testing.T) {
 	tests := []struct {
 		name                      string
 		rejectFilter              bool
+		failScore                 bool
 		rejectPostFilter          bool
 		expectFilterNumCalled     int
+		expectScoreNumCalled      int
```

Comment:
It seems you didn't verify this?

BTW: please rebase the code, there is some change in the test.

Reply:
And due to recent change, `sp.numScoreCalled` should be obtained in an atomic manner: `if atomic.LoadInt32(&scorePlugin.numScoreCalled) blabla...`.

Please help to update L751 as well.

Reply:
> It seems you didn't verify this?

thanks, I added.

>  please rebase the code, there is some change in the test.

rebased.

> sp.numScoreCalled should be obtained in an atomic manner

fixed!